src/foam/nanos/boot/NSpec.js

```diff
@@ -118,9 +118,7 @@ foam.CLASS({
     {
       class: 'FObjectProperty',
       name: 'service',
-      view: 'foam.u2.DetailView',
-      // Why was this set to be a SectionedDetailView? Put back to DetailView. KGR
-      //      view: { class: 'foam.u2.detail.SectionedDetailView' },
+      view: 'foam.u2.view.FObjectView',
```

Comment:
Don't we need to allowCustom: true for this?

Reply:
I don't believe we need to explicitly pass on a boolean for allowCustom since FObjectView automatically computes the boolean based on the length of this.choices via expression.